tests/functional/spec/textfield_suggestions.spec.js

```diff
@@ -8,3 +9,22 @@ describe("Suggestions", () => {
     expect($$(".js-autosuggest-listbox li").length).to.not.equal(0);
   });
 });
+
+describe("Suggestions", () => {
+  it("Given I open a textfield with a suggestions url, when I have entered text and picked suggestion from a list, then after typing more text it will show new suggestions", () => {
```

Comment:
This should mention it allows multiple suggestion.

Reply:
Added to description, thanks!